importer/CoEs.py

```diff
@@ -0,0 +1,87 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class CoEs(Monument):
+
+    def set_heritage_id(self):
+        """
+        Set the WLM ID as heritage ID.
+
+        The ID numbers in this dataset
+        are not real.
+        Note that since the WLM identifiers are not
+        globally unique, this might need to be
+        prefixed by country if convention is laid down.
+        """
+        self.add_statement("wlm_id", str(self.id))
+
+    def update_descriptions(self):
+        descs = {
+            "en": "national monument of Colombia",
+            "es": "Monumento Nacional de Colombia"
+        }
+        for lg in descs:
+            self.add_description(lg, descs[lg])
+
+    def set_monuments_all_id(self):
+        """Map which column name in specific table to  ID in monuments_all."""
+        self.monuments_all_id = str(self.id)
+
+    def update_labels(self):
+        spanish = utils.remove_markup(self.monumento)
+        self.add_label("es", spanish)
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        First try to resolve the municipality,
+        and if that doesn't work use the department
+        which is a higher unit.
+        """
+        match = None
+        dep_dic = self.data_files["departments"]
+        if self.has_non_empty_attribute("municipio"):
+            match = utils.q_from_first_wikilink("es", self.municipio)
+        else:
```

Comment:
Currently if we fail to match municipio we don't add anything at all. Would it be worth adding department in that case (in addition to reporting it).

i.e.
```python
match = None
dep_dic = self.data_files["departments"]
if self.has_non_empty_attribute("municipio"):
    match = utils.q_from_first_wikilink("es", self.municipio)
    if not match:
        self.add_to_report("municipio", self.municipio, "located_adm")
if not match:
    dep_match = [x for x in dep_dic if x["iso"] == self.iso]
    if len(dep_match) == 1:
        match = dep_match[0]["item"]
if match:
    self.add_statement("located_adm", match)
```

Reply:
Done